packages/filecmd.rb

```diff
@@ -53,5 +64,6 @@ def self.install
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin/"
     FileUtils.install 'filefix', "#{CREW_DEST_PREFIX}/bin/filefix", mode: 0o755
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install-strip'
+    FileUtils.cp 'file.static', "#{CREW_DEST_PREFIX}/bin/file"
```

Comment:
Should this be `FileUtils.install 'file.static', "#{CREW_DEST_PREFIX}/bin/file", mode: 0o755` to make sure it is executable.

Reply:
Done. (But the file binary is also already made executable by make.)